lib/modules/dosomething/dosomething_reportback/includes/ReportbackItem.php

```diff
@@ -115,8 +115,15 @@ private function build($data) {
         'verb' => $data->verb,
       ],
     ];
+
+    $northstar_user = dosomething_northstar_get_northstar_user($data->uid);
```

Comment:
can you `json_decode` the whole thing here, so you don't have to do it for each thing below?